js/main.js

```diff
@@ -1,17 +1,47 @@
-function handleSubmit(event)
-{
+function handleSubmit(event) {
   event.preventDefault();
   var movieName = document.querySelector('.title').value;
-  getMovieData(movieName.split(" ").join("+"));
+  getMovieData(movieName.split(' ').join('+'));
+  switchViews('movie-view');
 }
-document.querySelector('form').addEventListener('submit',handleSubmit);
+document.querySelector('form').addEventListener('submit', handleSubmit);
 
 function getMovieData(name) {
   var xhr = new XMLHttpRequest();
+  var movieData;
   xhr.open('GET', 'https://www.omdbapi.com/?apikey=d0b53caa&t=' + name);
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
-    console.log(xhr.response);
+    addMovie(xhr.response);
   });
   xhr.send();
+  return movieData;
+}
+
+function addMovie(movieObject) {
+  document.querySelector('.movie-title').textContent = movieObject.Title;
+  document.querySelector('.year').textContent = movieObject.Year.split('–').join(' – ');
+  document.querySelector('.genre').textContent = movieObject.Genre;
+  document.querySelector('.imdb-rating').textContent = movieObject.imdbRating;
+  document.querySelector('.plot').textContent = movieObject.Plot;
+  document.querySelector('.poster').setAttribute('src', movieObject.Poster);
+}
```

Comment:
Instead of editing elements already existing in the HTML document, you should make a function to generate this DOM tree for a single movie, then `return` that DOM tree so you can append it to the container it would be in when the function is called. That way if you ever need to use a similar DOM tree elsewhere in the app, you don't need to write a bunch of new code, you can just reuse that function.